.github/workflows/build-push.yaml

```diff
@@ -0,0 +1,44 @@
+name: build (push)
+
+on:
+  push:
+    branches: [ "main", "release-*" ]
+    tags:
+      - '^v?\d+\.\d+\.\d+(?:-.*)?$'
+
+permissions:
+  id-token: write # This is required for requesting the JWT token
+  contents: read # This is required for actions/checkout
+
+jobs:
+  compute-tags:
+    runs-on: ubuntu-latest
+    outputs:
+      tags: ${{ steps.get_tag.outputs.TAGS }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Get the latest tag
+        id: get_tag
+        run: |
+          {
+            echo 'TAGS<<EOF'
+            echo "${{ github.sha }}"
+            echo "${{ github.ref_name}}"
+            echo EOF
+          } >> "$GITHUB_OUTPUT"
+
+  build-operator:
+    needs: compute-tag
+    uses: kyma-project/test-infra/.github/workflows/image-builder.yml@main # Usage: kyma-project/test-infra/.github/workflows/image-builder.yml@main
+    with:
+      name: dockerregistry-operator
+      dockerfile: components/operator/Dockerfile
+      tags: ${{ needs.compute-tag.outputs.tag }}
```

Comment:
Shouldn't it be `needs.compute-tag.outputs.tags` instead of `needs.compute-tag.outputs.tag`?